Source/Request.swift

```diff
@@ -1001,6 +1057,19 @@ open class UploadRequest: DataRequest {
 
         return stream
     }
+
+    open override func cleanup() {
```

Comment:
I like this separation. I think it'll play into a formalization of lifetime event's we'll need to do to enable arbitrary subclasses.